src/components/SandboxBlock/SandboxBlock.tsx

```diff
@@ -22,248 +23,250 @@ import {useTranslation} from 'next-i18next';
+        const rtlIcon = isRtl ? TextAlignRight : TextAlignLeft;
 
-    const rtlIcon = isRtl ? TextAlignRight : TextAlignLeft;
+        const handleThemeChange = React.useCallback(() => {
+            setIframeTheme((theme) => (theme === 'dark' ? 'light' : 'dark'));
+        }, []);
 
-    return (
-        <div className={b({'full-screen': isFullScreen})}>
-            <Row space="0">
-                <Col s="12" l="8" m="8" className={b('wrapper-iframe')}>
-                    <Spin size="s" />
-                    <iframe
-                        ref={iframeRef}
-                        src={window && `${window?.location.origin}/sandbox/${libId}/${componentId}`}
-                        frameBorder={0}
-                        className={b('iframe')}
-                    />
-                </Col>
-                <Col s="12" l="4" m="4">
-                    <div className={b('top-actions')}>
-                        <div className={b('top-actions-wrapper')}>
-                            <Tooltip content={t('theme')}>
-                                <div
-                                    tabIndex={0}
-                                    role="button"
-                                    className={b('control-icon')}
-                                    onClick={() => {
-                                        setIframeTheme(iframeTheme === 'dark' ? 'light' : 'dark');
-                                    }}
-                                >
-                                    <Icon data={themeIcon} size={18} />
-                                </div>
-                            </Tooltip>
+        const handleDirectionChange = React.useCallback(() => {
+            setIframeDirection((direction) => (direction === 'ltr' ? 'rtl' : 'ltr'));
+        }, []);
+
+        const handleFullScreenChange = React.useCallback(() => {
+            setIsFullScreen((isFullScreenLocal) => !isFullScreenLocal);
+        }, []);
 
-                            {isSupportRTL && (
-                                <Tooltip content={isRtl ? t('rtlOff') : t('rtlOn')}>
+        return (
+            <div className={b({'full-screen': isFullScreen})}>
+                <Row space="0">
+                    <Col s="12" l="8" m="8" className={b('wrapper-iframe')}>
+                        <Spin size="s" />
+                        {isClient ? (
+                            <iframe
+                                ref={iframeRef}
+                                src={
+                                    window &&
+                                    `${window?.location.origin}/sandbox/${libId}/${componentId}`
+                                }
+                                frameBorder={0}
+                                className={b('iframe')}
+                                onLoad={handleIframeLoad}
+                            />
+                        ) : (
+                            <Skeleton className={b('iframe')} />
```

Comment:
Some issues with styles when the skeleton is showing:

https://github.com/user-attachments/assets/3ebba115-fa54-4da5-88b3-d9aafbd1428e

Reply:
But iframe don't loaded yet, I don't see alert component in iframe.

Reply:
Fixed incorrect col size